[Quazip] music does not compile on linux

I tried a compilation from scratch (removed external projects and the build directory) of https://github.com/Inria-Asclepios/medInria-public/pull/606 rebased on music3 from this morning. It does not compile at the quazip step:

```
[ 87%] Performing configure step for 'quazip'
Cannot find file: /home/mathildemerle/Dev/MUSIC4/music/medInria-public_ExtProjs/src/quazip/quazip.pro.
make[2]: *** [superbuild/CMakeFiles/quazip.dir/build.make:108 : /home/mathildemerle/Dev/MUSIC4/music/medInria-public_ExtProjs/build/Stamp/quazip/quazip-configure] Erreur 2
make[1]: *** [CMakeFiles/Makefile2:1527 : superbuild/CMakeFiles/quazip.dir/all] Erreur 2
make: *** [Makefile:152 : all] Erreur 2
```

Edit: my build quazip directory is empty. The src one has the quazip code.

--- a/superbuild/projects_modules/quazip.cmake
+++ b/superbuild/projects_modules/quazip.cmake
@@ -25,7 +25,7 @@ if (NOT USE_SYSTEM_${ep})
 ## #############################################################################
 
 set(git_url ${GITHUB_PREFIX}stachenov/quazip.git)
-set(git_tag master)
+set(git_tag v0.9.1)
 
 ## #############################################################################
 ## Add specific cmake arguments for configuration step of the project

